.pre-commit-config.yaml

```diff
@@ -0,0 +1,45 @@
+#exclude: |
+#  (?x)(
+#    ^testcases/testcase1/models/wrapped.mo|
+#    ^testcases/testcase2/models/wrapped.mo|
+#    ^testcases/testcase3/models/wrapped.mo|
+#    ^testing/references/parser/wrapped.mo
+#  )
```

Comment:
Is there a reason these are still present but commented?

Reply:
I was originally not touching these files until I figured out why they were failing in the tests (no hard line return at end). I left them in here commented to show the syntax since it is abnormal IMO.

Reply:
Ok sounds good, thanks.